materialsOptical.xml

```diff
@@ -66,9 +115,8 @@
 </material>
 
 <material Z="19" name="Photocathode" state="solid">
-      <property name="EFFICIENCY" ref="Cathode_Surf_Efficiency"/>
```

Comment:
PMT efficiency shows up in the matrices.xml output files where a skin-surface volume is created, not in the bulk-material properties